src/blocks/gallery-collage/styles/style.scss

```diff
@@ -12,6 +12,10 @@
 		list-style-type: none;
 		margin: 0;
 		padding: 0;
+
+		li {
+			margin: inherit;
+		}
```

Comment:
In terms of targeting only `.is-twentytwenty`, I am not sure that we should limit in this case. `table-layout: auto` seemed to render correctly in the different themes. What are your thoughts?